e2e/smoke_test.go

```diff
@@ -76,39 +81,7 @@ var _ = Describe("Smoke End To End Tests - against OpenShift Cluster with Istio
 							"--run", "ruby ratings.rb 9080",
 							"--route", "header:x-test-suite=smoke",
 							"--session", sessionName,
-						)
```

Comment:
What happened here?

Reply:
Removed the test which is redundant. This is covered by `DeploymentConfig` anyway (implicit namespace).

Reply:
Now we have Deployment testing without NS and DeploymentConfig testing with NS? That would technically leave k8s without any tests for specific ns...?

Not the biggest chance of error in the world I guess but

Reply:
Quite the opposite. We test `Deployment` with specified ns as we use kubectl for that, and current namespace/project (or implicit ns) for `DeploymentConfig`. Or am I missing sth?

Reply:
opposite, right still 1 option not test on k8s. but meh..